### PR TITLE
esummers - finished lab work (per Kate V.'s request)

### PR DESCRIPTION
**Pull request by esumm**

### CODE REVIEW

**Comment by talentedmrjones** [comment]
Hi Eric thanks for finishing this. I will leave some feedback as inline comments

File: www/index.html [hunk 1]
```diff
@@ -22,17 +22,25 @@
         <![endif]-->
 
         <div id="details">
-            <img id="title" src="img/card-title.jpg" alt="MDC Partners - Polaroid Celebration" />
-            <img id="description" src="img/card-description.jpg" alt="Instant - The Story of Polaroid" />
-            <img id="date" src="img/card-date.jpg" alt="October 30th - The MDC Innovation Centre, 745 5th Ave., 19th Floor" />
+            <img id="title" src="img/card-title.png" alt="MDC Partners - Polaroid Celebration" />
+            <img id="description" src="img/card-description.png" alt="Instant - The Story of Polaroid" />
+            <img id="date" src="img/card-date.png" alt="October 30th - The MDC Innovation Centre, 745 5th Ave., 19th Floor" />
```
Comment (talentedmrjones) [inline www/index.html:27]: I'm not seeing the improvement to the semantic structure as the instructions mentioned. There is an awful lot of content here in the `alt` attributes not defined (wrapped) by more semantic (meaningful) elements. Why leave this content as imagery?

File: www/index.html [hunk 2]
```diff
@@ -22,17 +22,25 @@
...
         </div>
         <div id="invitation">
             <h1>Stop by for a minute.</h1>
```
Comment (talentedmrjones) [inline www/index.html:30]: Does this really qualify as a heading? What is the basis for your decisions when choosing headings?

File: www/js/main.js [hunk 3]
```diff
@@ -1 +1,32 @@
...
+
+$('a.modal_close').on('click', function(){
+	closeModal();
+});	
+
+
+
+// jQuery modal functions
+
+function openModal(text1, text2) {
```
Comment (talentedmrjones) [inline www/js/main.js:22]: How do you justify using the global namespace?

Comment (esumm) [inline www/js/main.js:22]: hi Richard,

Thanks for the feedback.  Here's some answers to your questions:
1. _improvement on semantic structure -_ _When I saw the image alt-text, I
   thought that it was acceptable for SEO or screen-readers, as both will
   actively seek-out that alt-text.  I've often resisted adding elements to
   'create' semantic markup, as I've oftentimes worked within CMS's where you
   have no flexibility to allow for additional elements, so using alt-text
   seemed like my only ally.  SEO and Accessibility standards are an
   ever-changing bag-of-tricks that I've tried to keep an eye on, but I'm open
   for any suggestions that will help improve this knowledge._
2. _does the existing <h1> qualify as a heading - *_The <h1> tag slipped by
   me.  Generally, I am looking for one <h1> tag on a page (usually an easy
   go-to is the logo-space, where I'll wrap the logo image in an <h1>).  This
   was the only <h1> used on the page, so I didn't question it's usage based
   on it's content.  I agree, at this point the <h1> is simply font-formatting
   as the text "Stop by for a minute" isn't terribly telling about the page or
   it's purpose.*

3.\* global namespace 'mucking' -*\* It only occurred to me after the fact
that I could have wrapped this in a named object\* _to protect it from the
global namespace_. _I understand and get the practice, but am more-used to
working with and creating isolated variables/functions in much larger
projects.  Being the only custom jquery for this 'project', my goal was to
keep it as simple as possible, hence this was relegated to an afterthought._

I'm curious on 'the ideal fix' you were looking for on problems #1&2.  At
my previous post at VML (vml.com), I had the pleasure of working with some
very smart SEO folks.  I'm somewhat surprised they let me get away with
what I did on Kellogg's and SAP! (_no <h1> included_).  Looking back, I see
certain ways I could've made these sites better, from an SEO and
accessibility standpoint.  I've been locked-in a front-end-developer role
for six years, but I'm always trying to expand the knowledge-base on things
that are in my peripheral.

Thanks again for the opportunity!

eric

Eric Summers
esumm.webdev@gmail.com
www.esumm.com

**Comment by talentedmrjones** [comment]
Hi Eric, thanks again for completing this! I appreciate you taking the time and showing your interest in CPB.
~Richard

**Pull request closed without merging**